cartoframes/data/clients/bigquery_client.py

```diff
@@ -32,31 +32,43 @@ def wrapper(self, *args, **kwargs):
 
 class BigQueryClient(object):
 
-    def __init__(self, project, credentials):
-        self._project = project
+    def __init__(self, credentials):
         self._credentials = credentials or get_default_credentials()
-        self._bucket = 'carto-do-{username}'.format(username=self._credentials.username)
-        self.bq_client, self.gcs_client = self._init_clients()
+        self.bq_client = None
+        self.gcs_client = None
+
+        self.public_data_project = None
+        self.user_data_project = None
+        self.dataset = None
+        self.bucket_name = None
+        self.instant_licensing = None
+
+        self._init_clients()
 
     def _init_clients(self):
-        google_credentials = GoogleCredentials(self._credentials.get_do_token())
+        do_credentials = self._credentials.get_do_credentials()
+        google_credentials = GoogleCredentials(do_credentials.access_token)
 
-        bq_client = bigquery.Client(
-            project=self._project,
+        self.bq_client = bigquery.Client(
+            project=do_credentials.gcp_execution_project,
```

Comment:
`bq_client` uses `gcp_execution_project` and `gcs_client` uses `bq_project`. Is this right?

Reply:
Yes, it is